### DISCUSSION TITLE
Pre-built Docker image for linux/arm64/v8 platform

### DISCUSSION THREAD

**Discussion by alexelgt** [ente-io/ente#1305] [Enhancements]
Hi, I followed this [guide for using pre-built Docker images](https://github.com/ente-io/ente/blob/main/server/docs/docker.md) and I get this error when I put the command `docker compose up`:

`museum The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested`

Could it be possible to add an image for linux/arm64/v8?

**Comment by mnvr** [comment]
Hey, right now we'd recommend just building the docker image from source if the pre-built image is not sufficient for your architecture. It is quite simple really.

```sh
git clone https://github.com/ente-io/ente
cd ente/server
docker compose up --build
```

But yes, down the line when we revisit this we'll try to publish images for more architectures. Thanks!

**Comment by mnvr** [comment]
We just published a new package version that support `linux/amd64`, `linux/arm64` and `linux/arm/v7`. So now the pre-built images should work for your architecture too.

https://github.com/ente-io/ente/pkgs/container/server

**Reply by mnvr** [reply]
Note that if you have a cached image on your machine, you'll need to get docker to pull the latest. e.g. 
```sh
docker compose up --pull always
```

Or you can just pull the latest image specifically for Ente's server by
```sh
docker pull ghcr.io/ente-io/server
```

**Reply by alexelgt** [reply]
Thank you got it working with the pre-built image

**Reply by mnvr** [reply]
Awesome! Thank you for giving it a go 🙏
